internal/ignition/dummy.go

```diff
@@ -0,0 +1,96 @@
+package ignition
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/assisted-service/pkg/s3wrapper"
+	"github.com/sirupsen/logrus"
+)
+
+const kubeconfig string = `
+clusters:
```

Comment:
Can this yaml be defined on a separated `ignition/kubeconfig.yaml` file?

Reply:
It could, but it is not clear to me what would be the benefit of having it in a separate file. If it is separated, in Dockerfile.assisted-service, ignition/kubeconfig.yaml would need to be copied to the container image so that it is accessible during runtime. Having to remember why we have to copy this file in the Dockerfile, I think, complicates long term maintenance.